Royalty/Patches/HeDiffDefs/Hediffs_Implants.xml

```diff
@@ -208,7 +208,7 @@
 	<Operation Class="PatchOperationReplace">
 		<xpath>Defs/HediffDef[defName="StoneskinGland"]/stages/li/statOffsets/ArmorRating_Sharp</xpath>
 		<value>
-			<ArmorRating_Sharp>5</ArmorRating_Sharp>
+			<ArmorRating_Sharp>10</ArmorRating_Sharp>
```

Comment:
I'd drop this to about 6 - 8, making them reasonably durable against gun fire up to 7.62 NATO (FMJ), but without being able to soak up so much of it or being entirely beyond the reach of non-spacer melee weapons like a good halberd.